testsuite/integration-arquillian/tests/pom.xml

```diff
@@ -1693,6 +1697,20 @@
             </properties>
         </profile>
 
+        <profile>
+            <id>java11-auth-server</id> <!-- a temporary workaround; TODO remove this once Java 11 is officially supported by Arquillian -->
```

Comment:
Wouldn't be possible to activate the profile automatically using
```suggestion
            <id>java11-auth-server</id> <!-- a temporary workaround; TODO remove this once Java 11 is officially supported by Arquillian -->
            <activation>
                <jdk>11</jdk>
            </activation>
```
to avoid the need to select the profiles manually?

Reply:
I'm not sure if is is 11 or e.g. 1.11, as far as I was able to find out it's derived from system property java.version [1]

[1] https://github.com/apache/maven/blob/maven-3.6.0/maven-model-builder/src/main/java/org/apache/maven/model/profile/activation/JdkVersionProfileActivator.java#L63

Reply:
As far as I know, It's not possible, unfortunately. At normal conditions, we don't run the tests as such (i.e. `mvn` command) under JDK 11, only the auth/app server is using that JDK. Or am I wrong?

Reply:
I mean, the solution you suggested would work only when mvn JDK == server JDK.

Reply:
@vmuzikar you're right, thank you